File: java/ex3.java
import java.util.Scanner;

public class ex3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N;
        N = scanner.nextInt();

        int sum = 0;
        int product = 1;
        while (N > 0) {
            sum += N % 10;
            product *= N % 10;
            N /= 10;
        }
        System.out.println(sum + " " + product + " " + Math.abs(sum - product));
        scanner.close();
    }
}

File: Kotlin/ex3.kt
import java.util.Scanner
fun main() {
    val scanner = Scanner(System.`in`)
    var N: Int
    N = scanner.nextInt()

    var sum = 0
    var product = 1
    while (N > 0) {
        sum += N % 10
        product *= N % 10
        N /= 10
    }
    println("$sum $product ${Math.abs(sum - product)}")
}